src/controllers/rooms.controller.js

```diff
@@ -0,0 +1,73 @@
+const { Room } = require('../models/room.model');
+const roomsService = require('../services/rooms.service');
+const { ApiError } = require('../utils/api.error');
+
+const getAll = async (req, res) => {
+  const rooms = await roomsService.getAll();
+
+  res.send(rooms);
+};
+
+const create = async (req, res) => {
+  const { name, userId } = req.body;
+
+  if (!name || !userId) {
+    throw ApiError.badRequest('Not all data is provided!');
+  }
+
+  const possibleRoom = await Room.findOne({ where: { name } });
+
+  if (possibleRoom) {
+    throw ApiError.badRequest('This room is already exists');
```

Comment:
There's a grammatical error in the error message: 'This room is already exists'. It should be 'This room already exists'.